pasta_eln/GUI/form.py

```diff
@@ -170,9 +170,10 @@ def __init__(self, comm:Communicate, doc:dict[str,Any]):
       self.formL.addRow(QLabel('Data type'), self.docTypeComboBox)
     #final button box
     _, buttonLineL = widgetAndLayout('H', mainL)
-    visibilityIcon = all(all(branch['show']) for branch in self.doc['-branch'])
-    self.visibilityText = QLabel('' if visibilityIcon else 'HIDDEN     \U0001F441')
-    buttonLineL.addWidget(self.visibilityText)
+    if '-branch' in self.doc:
```

Comment:
I have seen that many places in form.py, the same access (self.doc['-branch']) been done will this scenario arise in those code blocks too? Also are we sure that the else case in this scenario be addressed without any further regression?

Reply:
@jmurugan-fzj 
- yes, we can be sure that it (no -branch) arises in few cases: new documents. -branch, just as -name and -tags starts with a - and is hence essential: all documents in the database have it; only ones that are on the way to the database (aka new), potentially don't have it. Hence, the "add" button and the form has worked for a long time without issues. Only, adding the "HIDDEN" information has triggered the bug. So taking out the "Hidden" information for new documents, seems logical
- the else case: is not display the "hidden" text; a new item cannot be hidden.